macros/src/main/scala/code/chymyst/jc/Macros.scala

```diff
@@ -169,13 +169,16 @@ final class BlackboxMacros(override val c: blackbox.Context) extends ReactionMac
 
     // Note: `caseDefs` should not be an empty list because that's a typecheck error (`go` only accepts a partial function, so at least one `case` needs to be given).
     // However, the user could be clever and write `val body = new PartialFunction...; go(body)`. We do not allow this because `go` needs to see the entire reaction body.
-    if (caseDefs.isEmpty) reportError("No `case` clauses found: Reactions must be defined inline with the `go { case ... => ... }` syntax")
+    if (caseDefs.isEmpty)
```

Comment:
Would you consider necessary/useful a pattern match on caseDefs that avoids making use of length?